src/server/middleware/html.jsx

```diff
@@ -33,6 +34,7 @@ const Html = ({ content, state, assetMap, css, helmet, token, refreshToken }) =>
           />
         )}
         {!!css && css}
+        {settings.subscription.enabled && <script src="https://js.stripe.com/v3/" />}
```

Comment:
Try and move this to module and abstract it similarly as `stylesInserts` is doing for styles.

Reply:
Great idea. Added.